content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,439 @@
+
+When a check is executed and an event is generated, Sensu splits handling
+of the event for each handler defined. When multiple handlers are defined, each
+handler is managed independently. This means that each handler is processed
+independent of any other handler configured.
+
+<img alt="Contact Routing and Monitoring Event Pipeline" title="Contact Routing and Monitoring Event Pipeline" src="/images/contact-routing-MEP.png">
+
+Contacts are merged into the check handlers after filters are applied. After merging in the different valid contacts, a handler is executed for each valid contact. If a contact does not have a configuration for a handler, then the default configuration for that handler is used instead.
+
+## Contact Configuration
+
+Configuring a contact requires you to define the name of the contact and the
+services plus configuration override for that contact.
+
+{{< highlight json >}}
+{
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In the example above we have a contact named **support**. For **support** we
+have an override configuration for email and slack. In our example we're having
+this contact email _support@sensuapp.com_ instead of the email intergration's
+default **to** address. For Slack, we're defining the channel the event should
+post to, in this case _#support_.
+
+## Check Configuration
+
+Once a contact has been defined, we can now apply which contact to use in a
+check configuration. The use case for having it on a check could be that the
+client is owned by a particular team or group, the same can be done for specific
+checks. For instance 
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handler": "email",
+      "contact": "support"
+    }
+  }
+}
+{{< /highlight >}}
+
+## Client Configuration
+
+A client configuration can include a contact to use.
+
+{{< highlight json >}}
+{
+  "client": {
+    "name": "ec2-west-id-12345",
+    "address": "8.8.8.8",
+    "subscriptions": [
+      "production",
+      "aws"
+    ],
+    "contact": "aws-team"
+  }
+}
+{{< /highlight >}}
+
+# Example Implementations
+
+## Default Configuration
+
+For the examples we'll be working with we want to define what our default handlers will be.
+
+In the following section we'll be going over different scenarios and showcase how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and Pagerduty as our example handlers.
+
+Bellow is the default configuration for the handlers we'll be working with in our examples:
+
+{{< highlight json >}}
+{
+  "handlers": {
+    "slack": {
+      "webhook_url": "https://hooks.slack.com/services/foo/bar/foobar",
+      "username": "sensu",
+      "channel": "#alerts",
+      "timeout": 10
```

Comment:
This might be confusing since the Slack integration doesn't appear within the handler scope in the Sensu Enterprise configuration. I'm not sure about email and PagerDuty.

Reply:
removed the "handler" scope as it's an integration 🤦‍♂️